configs/rt_slam_plugins.yaml

```diff
@@ -3,14 +3,14 @@ plugin_group:
   # - path: zed/
   # - path: realsense/
   # - path: depthai/
-  - name: Kimera-VIO/
```

Comment:
Let's keep Kimera instead of OpenVINS in this PR. We will need to make both work, but still leave Kimera as the default.

Reply:
Other than that, looks good!

Reply:
Added Kimera back and commented OpenVINS/